src/main/java/com/thullo/service/TaskServiceImpl.java

```diff
@@ -221,6 +221,12 @@ public List<Task> editStatus(StatusRequest request) throws ResourceNotFoundExcep
         return taskRepository.saveAll(tasks);
     }
 
+    @Override
+    public List<Task> deleteStatus(StatusRequest statusRequest) throws ResourceNotFoundException {
+        statusRequest.setStatus(formatStatus("NO_STATUS"));
```

Comment:
You can parse the string directly without the need to call the formatStatus method. Additionally, when will you have a chance so that we can quickly show you how to document on Postman?